src/System.Json/src/System.Json.csproj

```diff
@@ -3,6 +3,10 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <ProjectGuid>{B11B5B76-F583-44EA-A4B7-3ABD990594F6}</ProjectGuid>
+    <!-- 
+        System.Json mainly provide compatibility for Mono apps. we should keep the identity of this library matching 
```

Comment:
It isn't just about Mono. I would say this is a library for compat purposes so we want to keep the same identity as it originally shipped with in SL.